interfaces/ISO15118_charger.yaml

```diff
@@ -175,149 +175,149 @@ cmds:
   reset_error:
     description: Reset all errors
 vars:
-  Require_Auth_EIM:
+  require_auth_eim:
     description: An EIM authorization is requiered
     type: "null"
-  Require_Auth_PnC:
+  require_auth_pnc:
     description: >- 
       The EVCC provides the payment details for a PnC authorization by sending
       the signature certificate chain and eMAID.
     type: object
     $ref: /authorization#/ProvidedIdToken
-  AC_Close_Contactor:
+  ac_close_contactor:
     description: The contactor should be closed
     type: "null"
-  AC_Open_Contactor:
+  ac_open_contactor:
     description: The contactor should be opened
     type: "null"
-  Start_CableCheck:
+  start_cable_check:
     description: The charger should now start a cable check
     type: "null"
-  DC_Open_Contactor:
+  dc_open_contactor:
     description: The contactor should be opened
     type: "null"
-  V2G_Setup_Finished:
+  v2g_setup_finished:
     description: >-
-      V2G_Setup_Finished from ISO15118-3. Trigger when EV sends a PowerDeliveryReq
+      v2g_setup_finished from ISO15118-3. Trigger when EV sends a PowerDeliveryReq
       message with ChargeProgess equals Start or Stop
     type: "null"
-  currentDemand_Started:
+  current_demand_started:
     description: The charging process has started and the EV wants to be charged
     type: "null"
-  currentDemand_Finished:
+  current_demand_finished:
     description: The charging process was finished
     type: "null"
   sae_bidi_mode_active:
       description: The SAE J2847 bidi mode is active
       type: "null"
-  EVCCIDD:
+  evccidd:
     description:
       Specifies the EVs identification in a readable format. It contains
       the MAC address of the EVCC in uppercase
     type: string
     pattern: ^[A-F0-9]{2}(:[A-F0-9]{2}){5}$
-  SelectedPaymentOption:
+  selected_payment_option:
     description: This element is used for indicating the payment type
     type: string
     $ref: /iso15118_charger#/PaymentOption
-  RequestedEnergyTransferMode:
+  requested_energy_transferMode:
```

Comment:
shouldn't it be transfer_mode?

Reply:
Thx :) this is correct. I'll change that